### DISCUSSION TITLE
Where to find mmdet.apis documentation?

### DISCUSSION THREAD

**Discussion by MeAmarP** [open-mmlab/mmdetection#9631] [General]
Hi coders,
Am **_unable to find the documentation for mmdet.apis_**. I was going through the inference demo notebook which has import statement.
```python
from mmdet.apis import init_detector, inference_detector, show_result_pyplot
```

Now if you refer the docs at  [https://mmdetection.readthedocs.io/en/latest/api.html#](https://mmdetection.readthedocs.io/en/latest/api.html#), I could not find any. only option is to go to actual implementation of the code. but there again documentation is not much verbose or detailed. refer screen-shot below.
![image](https://user-images.githubusercontent.com/12434567/212467565-0ea60494-55f5-4fa2-94b7-c60cf7e058fb.png)

Can you please help me find it? Thanks

**Comment by MeAmarP** [comment]
Moved this discussion to General, No response from team.

**Comment by Li-Qingyun** [comment]
https://mmdetection.readthedocs.io/en/latest/api.html
![1675137952890](https://user-images.githubusercontent.com/79644233/215660357-4ab11b4c-175d-4852-a2e1-acd1748f5396.png)
